build.py

```diff
@@ -1,38 +1,10 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from conan.packager import ConanMultiPackager
-from conans import tools
-import importlib
-import os
 
+from bincrafters import build_template_default
 
-def get_module_location():
-    repo = os.getenv("CONAN_MODULE_REPO", "https://raw.githubusercontent.com/bincrafters/conan-templates")
-    branch = os.getenv("CONAN_MODULE_BRANCH", "package_tools_modules")
-    return repo + "/" + branch
-
-    
-def get_module_name():
-    return os.getenv("CONAN_MODULE_NAME", "build_template_default")
-
-    
-def get_module_filename():
-    return get_module_name() + ".py"
-    
-    
-def get_module_url():
-    return get_module_location() + "/" + get_module_filename()
-
-    
 if __name__ == "__main__":
-    
-    tools.download(get_module_url(), get_module_filename(), overwrite=True)
-    
-    module = importlib.import_module(get_module_name())
-    
-    builder = module.get_builder()
-    
-    builder.run()
+    builder = build_template_default.get_builder(pure_c=False)
```

Comment:
I'm not sure about this as both Vorbis and Flac depend on this library, and it's unclear to me if this would cause an issue with them.

Reply:
Not sure if I understand this comment but this library is flac and exports symbols in C and CXX. it shouldn't be ```pure_c=True``` in CI.

I've done a better test_package using the c++ interface following [encode flac example.cpp](https://github.com/xiph/flac/blob/master/examples/cpp/encode/file/main.cpp) and it's tested in all modes (static/shared) and in all platforms.

Hope that's ok.